app/styles/style-flat.sass

```diff
@@ -158,7 +158,7 @@ body[lang='ru'], body[lang='uk'], body[lang='bg'], body[lang^='mk'], body[lang='
       padding: 30px 10px
       width: 100%
       
-    @media (max-width: $screen-lg-min)
+    @media (max-width: $screen-md-min)
```

Comment:
I'd just switch the above two instances of this to use `$grid-float-breakpoint` if they're meant to be kept in sync with that variable anyway.